Contact lists Fixes - TX and Contacts screens

monikasmolarek opened the pull request:
fixes #1132 

- [x]  [font color in the autocomplete dropdown](https://github.com/spacemeshos/smapp/issues/1132#issuecomment-1426910701)
- [ ] [sorting contacts in the autocomplete dropdown (tx screen)](https://github.com/spacemeshos/smapp/issues/1132#issue-1578378165)
- [x] [z-index]( https://github.com/spacemeshos/smapp/issues/1132#issuecomment-1460875232)
- [ ] [addresses abbreviation in some cases](https://github.com/spacemeshos/smapp/issues/1132#issuecomment-1460933119)
- [ ] [scrolling the autocomplete drop-down list](https://github.com/spacemeshos/smapp/issues/1132#issuecomment-1460960070)
- [x] [sorting the records on the Contacts screen](https://github.com/spacemeshos/smapp/issues/1132#issuecomment-1462908981)

Additionally, I added a tooltip on the Contacts screen, next to the "Name" field, informing about the "Send SMH" tx screen being opened by clicking on the contact nickname (which was not obvious/quite hidden, and I discovered it accidentally).

maparr commented:
Hi @monikasmolarek , when it will be done Click the Ready for review above. Thanks !

monikasmolarek commented:
Of course, until now, I was discovering more and more aspects to be covered, will be working on that, and once ready, I will mark it as ready and request your review

maparr committed:
feature: allow customisation of node config

dependabot[bot] committed:
chore(deps): bump throttle-debounce and @types/throttle-debounce

Bumps [throttle-debounce](https://github.com/niksy/throttle-debounce) and [@types/throttle-debounce](https://github.com/DefinitelyTyped/DefinitelyTyped/tree/HEAD/types/throttle-debounce). These dependencies needed to be updated together.

Updates `throttle-debounce` from 4.0.1 to 5.0.0
- [Release notes](https://github.com/niksy/throttle-debounce/releases)
- [Changelog](https://github.com/niksy/throttle-debounce/blob/master/CHANGELOG.md)
- [Commits](https://github.com/niksy/throttle-debounce/compare/v4.0.1...v5.0.0)

Updates `@types/throttle-debounce` from 4.0.0 to 5.0.0
- [Release notes](https://github.com/DefinitelyTyped/DefinitelyTyped/releases)
- [Commits](https://github.com/DefinitelyTyped/DefinitelyTyped/commits/HEAD/types/throttle-debounce)

---
updated-dependencies:
- dependency-name: throttle-debounce
  dependency-type: direct:production
  update-type: version-update:semver-major
- dependency-name: "@types/throttle-debounce"
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps): bump react-scroll from 1.8.7 to 1.8.9

Bumps [react-scroll](https://github.com/fisshy/react-scroll) from 1.8.7 to 1.8.9.
- [Changelog](https://github.com/fisshy/react-scroll/blob/master/CHANGELOG.md)
- [Commits](https://github.com/fisshy/react-scroll/commits)

---
updated-dependencies:
- dependency-name: react-scroll
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps): bump openpgp from 5.8.0 to 5.9.0

Bumps [openpgp](https://github.com/openpgpjs/openpgpjs) from 5.8.0 to 5.9.0.
- [Release notes](https://github.com/openpgpjs/openpgpjs/releases)
- [Commits](https://github.com/openpgpjs/openpgpjs/compare/v5.8.0...v5.9.0)

---
updated-dependencies:
- dependency-name: openpgp
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

maparr committed:
fix: update smeshing metadata and return empty object if no file exists

cl8n committed:
Use `setWindowOpenHandler()` in favor of deprecated `new-window` event

brusherru committed:
tweak: copy hex-encoded keys from Address component with leading `0x`

brusherru committed:
feat: display current node activity on smesher screen

brusherru committed:
feat: add node events log screen

brusherru committed:
feat: use proper HRP network defined by node config

brusherru committed:
tweak: change the default to HRP.MainNet

brusherru committed:
tweak: disable Wallet Only mode buttons

maparr committed:
fix: dont hide back your wallet after the backup

brusherru committed:
feat: display green / grey / red bulbs in the events log

maparr committed:
fix: use genesisID in custom config name

brusherru committed:
tweak: make HRP works in standalone mode

brusherru committed:
Release v0.3.15

brusherru committed:
fix: do not show "Switch network" twice on unlocking

brusherru committed:
fix(Version): stop the timer on unmounting component

maparr committed:
feat_1310: fix sending to same addreess

brusherru committed:
tweak: show "Node is preparing..." in empty Activity Log

brusherru committed:
fix: send some data from main process to renderer once again on unlocking wallet / switching network

brusherru committed:
chore: specify the correct type for `isSmeshing`

brusherru committed:
Release v1.0.0

maparr committed:
fix: redirect to Wallet from internal pages

maparr committed:
fix: error on show back button

maparr committed:
Release v1.0.1

monikasmolarek committed:
Update README.md

brusherru committed:
Release v1.0.2

brusherru committed:
fix: hard-code mainnet HRP

brusherru committed:
Release v1.0.3

brusherru committed:
fix: calculate genesis id properly

brusherru committed:
fix: calculate recommended size properly

maparr committed:
fix: update tests

brusherru committed:
fix: put `smeshing-opts-compute-batch-size` in a correct place in Config

brusherru committed:
tweak: add env-representation for app argument + added description into README.md

brusherru committed:
fix: properly check for changes in node config and don't restart node excessively

brusherru committed:
tweak(fetchDiscovery): add logging

brusherru committed:
fix: use the same directory for benchmarks as selected for PoS-data

brusherru committed:
Release v1.0.4

maparr committed:
fix: issue with ResizeObserver loop limit exceeded

dependabot[bot] committed:
chore(deps-dev): bump @storybook/react from 6.5.12 to 7.0.27

Bumps [@storybook/react](https://github.com/storybookjs/storybook/tree/HEAD/code/renderers/react) from 6.5.12 to 7.0.27.
- [Release notes](https://github.com/storybookjs/storybook/releases)
- [Changelog](https://github.com/storybookjs/storybook/blob/next/CHANGELOG.md)
- [Commits](https://github.com/storybookjs/storybook/commits/v7.0.27/code/renderers/react)

---
updated-dependencies:
- dependency-name: "@storybook/react"
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

maparr committed:
fix:  switch account, because of wrong away listener on isOpenedProp

maparr committed:
chore: update deps

maparr committed:
fix: send sentry attachments only for Feedback form

brusherru committed:
refactor: add `getShortGenesisId`, `getDefaultPosDir`, `getCustomNodeConfigPath`

brusherru committed:
feat: migrate genesis id: rename `node-data` dir and custom `node-config` & update wallet

brusherru committed:
feat: return back dynamic HRP

dependabot[bot] committed:
chore(deps): bump word-wrap from 1.2.3 to 1.2.4

Bumps [word-wrap](https://github.com/jonschlinkert/word-wrap) from 1.2.3 to 1.2.4.
- [Release notes](https://github.com/jonschlinkert/word-wrap/releases)
- [Commits](https://github.com/jonschlinkert/word-wrap/compare/1.2.3...1.2.4)

---
updated-dependencies:
- dependency-name: word-wrap
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
bump babel-loader from 8.2.2 to 9.1.3

brusherru committed:
feat: update API to v1.17.0 (EventInitFailed)

brusherru committed:
refactor: extract NodeEvent utils into separate module

brusherru committed:
tweak(PosProfiler): reduce SAFE_PERIOD coefficient from 0.8 to 0.7

brusherru committed:
Revert "feat: migrate genesis id: rename `node-data` dir and custom `node-config` & update wallet"

This reverts commit 19da36faa92e39ddc7ae26af47e0d3fe6c29460d.

brusherru committed:
Revert "fix: update tests"

This reverts commit 49789a97557b94590fb892ccb39b48e999866d2d.

brusherru committed:
Revert "fix: calculate genesis id properly"

This reverts commit c4615d2a9cfe71f64c2508f9e805956466d1821c.

brusherru committed:
Release v1.0.5

brusherru committed:
Release v1.0.7

maparr committed:
fix: add nonces and threads to redux

maparr committed:
fix: sync postProvingOpts with redux

maparr committed:
fix: do not clear smesher metadata if delete files false

brusherru committed:
fix: change the wrong wording for the `poetWaitProof` NodeEvent

dependabot[bot] committed:
rebase

Bumps [@storybook/react](https://github.com/storybookjs/storybook/tree/HEAD/code/renderers/react) from 7.0.27 to 7.1.1.
- [Release notes](https://github.com/storybookjs/storybook/releases)
- [Changelog](https://github.com/storybookjs/storybook/blob/next/CHANGELOG.md)
- [Commits](https://github.com/storybookjs/storybook/commits/v7.1.1/code/renderers/react)

---
updated-dependencies:
- dependency-name: "@storybook/react"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

brusherru committed:
Release v1.0.8

brusherru committed:
chore: update dependencies to get rid of critical audit warnings

brusherru committed:
chore: add new win.publisherName

brusherru committed:
feat: add env info to networks.json request

brusherru committed:
refactor: export useClickAway as default and get rid of lint warning

brusherru committed:
chore: update yarn.lock

brusherru committed:
fix: missed waiting time for `poetWaitProof` event

maparr committed:
feature: add CODE OF CONDUCT

maparr committed:
feature: add CONTRIBUTING

maparr committed:
feature: add SECURITY

brusherru committed:
Revert "chore: update yarn.lock"

This reverts commit 9c21e63d3924bc2fdee516f3b05624dfba440b31.

brusherru committed:
Revert "chore: update dependencies to get rid of critical audit warnings"

This reverts commit fb00350b243237510b5eb255e2c262e9278d2da9.

brusherru committed:
Release v1.0.9

dependabot[bot] committed:
chore(deps): bump auto-launch from 5.0.5 to 5.0.6

Bumps [auto-launch](https://github.com/4ver/node-auto-launch) from 5.0.5 to 5.0.6.
- [Release notes](https://github.com/4ver/node-auto-launch/releases)
- [Commits](https://github.com/4ver/node-auto-launch/compare/5.0.5...5.0.6)

---
updated-dependencies:
- dependency-name: auto-launch
  dependency-type: direct:production
  update-type: version-update:semver-patch
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps): bump fs-extra from 10.1.0 to 11.1.1

Bumps [fs-extra](https://github.com/jprichardson/node-fs-extra) from 10.1.0 to 11.1.1.
- [Changelog](https://github.com/jprichardson/node-fs-extra/blob/master/CHANGELOG.md)
- [Commits](https://github.com/jprichardson/node-fs-extra/compare/10.1.0...11.1.1)

---
updated-dependencies:
- dependency-name: fs-extra
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

brusherru committed:
Release v1.0.10

Commit:
fix: download progress bar too many decimals

Commit:
fix download progress bar too many decimals

brusherru committed:
feat: add `safeSmeshingProvingOpts` function and integrate into `safeSmeshingOpts`

brusherru committed:
refactor: get rid of excessive complexity

brusherru committed:
fix: remove invalid proving opts from custom config on loading custom config

brusherru committed:
test: fix tests

brusherru committed:
tweak: remove coinbase validation & conversion

brusherru committed:
Release v1.0.11

brusherru committed:
tweak: get rid of `no-cache` query string in requests to discovery

brusherru committed:
Release v1.0.12

brusherru committed:
fix: stop caching responses from discovery

maparr committed:
feat: add CI section, rename SMAPP, add contibutors to review this PR

maparr committed:
feature: add 24 words restore

maparr committed:
feature: add 24 words option during create wallet

maparr committed:
fix: update responsive design for test and backup pages

maparr committed:
fix: on recovery mode , don't ask to test recovery

maparr committed:
fix: handle wallet only mode and update naming

maparr committed:
feature: add link to latest release for Deb package

maparr committed:
fix: update height for backup section

maparr committed:
fix: define deb package, update naming, style for link

brusherru committed:
Release v1.0.13

maparr committed:
fix: move MnemonicStrengthType to shared/types/wallet.ts

maparr committed:
fix: add comment and don't pass mnemonic in case it is undefined on the SwitchNetwork screen

maparr committed:
feature: add app image to getEnvInfo

maparr committed:
fix: remove timer for AU_DOWNLOAD_MANUALLY handler

maparr committed:
fix: reuse isDebPackage

maparr committed:
fix: simplify checkUpdates

maparr committed:
fix: simplify autoUpdater statemtn for deb package

brusherru committed:
Release v1.0.14

PlainLazy committed:
sort eligible for rewards layers list

brusherru committed:
Release v1.0.15

brusherru committed:
fix(linter): sort eligible for rewards layers list

brusherru committed:
fix: get rewards by smesher coinbase correctly

brusherru committed:
fix: get coinbase from config instead of flaky API

brusherru committed:
refactor: remove excessive layer of retries

maparr committed:
fix: update isLinuxAppImage fn

maparr committed:
fix: remove scroll for 24 words backup and WordsBackup screens

brusherru committed:
fix: do not retrieve Activations to avoid high CPU/RAM usage

brusherru committed:
refactor: remove unused code

maparr committed:
fix: update interface for mnemonic exist and mnemonic generate for 12 / 24 words

brusherru committed:
refactor: extract some helpers into separate module to avoid circle imports

brusherru committed:
refactor: remove useless empty file

brusherru committed:
feat: log every fetch request and its response.status

brusherru committed:
tweak: hide wallet password in the app logs

brusherru committed:
test: mock electron-log

maparr committed:
fix: add constant for default mnemonicStrengthType create wallet fn

maparr committed:
fix: update color and size for test me words, to handle bug for stale step on drug and drop

brusherru committed:
Release v1.0.16

maparr committed:
fix: Windows not fit the global height and css calc fn broke pages sizes

dependabot[bot] committed:
chore(deps): bump protobufjs from 6.11.3 to 6.11.4

Bumps [protobufjs](https://github.com/protobufjs/protobuf.js) from 6.11.3 to 6.11.4.
- [Release notes](https://github.com/protobufjs/protobuf.js/releases)
- [Changelog](https://github.com/protobufjs/protobuf.js/blob/master/CHANGELOG.md)
- [Commits](https://github.com/protobufjs/protobuf.js/commits)

---
updated-dependencies:
- dependency-name: protobufjs
  dependency-type: indirect
...

Signed-off-by: dependabot[bot] <support@github.com>

brusherru committed:
Release v1.0.17

brusherru committed:
fix: show fatal errors on the network screen

brusherru committed:
fix: do not copy key.bin on changing pos dir

brusherru committed:
tweak: show up to two lines of error message on the Network screen

brusherru committed:
tweak: display the entire message on long hover

brusherru committed:
Release v1.1.1

maparr committed:
fix: update types for mnemonic

brusherru committed:
chore: update Application default paths in troubleshooting section

maparr committed:
Update proving opts if not exist (#1396)

feature: add reactions to handle error proving opts and update smeshing opts config

maparr committed:
feature: add update proving opts on the edit page

maparr committed:
feature: update title for recovery page and hide recovery feature on wallet screen when

maparr committed:
feature: add consistent update to specific version from networks json

maparr committed:
feature: add not supported version for network link to switch network

maparr committed:
feature: add responsive 24/12 words restore with consistent order

maparr committed:
feature: add process env system helpers and replace existing one

brusherru committed:
fix: render "genesis phase" properly and use verifiedLayer instead of syncedLayer

brusherru committed:
tweak: keep rendering "synced" status if verified layer is not more than 2 layers behind & isSynced: true

brusherru committed:
fix: determine `isGenesisPhase` by topLayer instead of verifiedLayer

maparr committed:
feature: add ensure node libraries installed on PC

maparr committed:
fix: reset node status on restart node

dependabot[bot] committed:
chore(deps): bump openpgp from 5.9.0 to 5.10.1

Bumps [openpgp](https://github.com/openpgpjs/openpgpjs) from 5.9.0 to 5.10.1.
- [Release notes](https://github.com/openpgpjs/openpgpjs/releases)
- [Commits](https://github.com/openpgpjs/openpgpjs/compare/v5.9.0...v5.10.1)

---
updated-dependencies:
- dependency-name: openpgp
  dependency-type: direct:production
...

Signed-off-by: dependabot[bot] <support@github.com>

brusherru committed:
fix: Wait for establishing GRPC connection without a timeout limit

brusherru committed:
fix: simplify retries for failed grpc requests/streams and do not log errors while connecting

maparr committed:
feature: add blockers and waiting messages for Smeshing screen

maparr committed:
fix: disable close button for delete pos files. Add description tooltips for start smeshing screen.

brusherru committed:
fix: use PPROF_SERVER env variable correctly (as a boolean switch)

brusherru committed:
fix: restart the Node properly

brusherru committed:
fix: updating node config and restarting the node when needed

brusherru committed:
fix: do not show error messages while Node is restarting

brusherru committed:
fix: remove special debouncing functions which does not handle properly delays inside callbacks

Can be optimized in a more effective and robust way.

brusherru committed:
tweak: Enlarge delay between retries for `isNodeAlive` (to spam less)

brusherru committed:
tweak: subscribe for account data when Node become responsive (or after timeout)

brusherru committed:
Release v1.1.3

dependabot[bot] committed:
chore(deps-dev): bump @storybook/addon-links from 6.5.12 to 7.4.0

Bumps [@storybook/addon-links](https://github.com/storybookjs/storybook/tree/HEAD/code/addons/links) from 6.5.12 to 7.4.0.
- [Release notes](https://github.com/storybookjs/storybook/releases)
- [Changelog](https://github.com/storybookjs/storybook/blob/next/CHANGELOG.md)
- [Commits](https://github.com/storybookjs/storybook/commits/v7.4.0/code/addons/links)

---
updated-dependencies:
- dependency-name: "@storybook/addon-links"
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

maparr committed:
feature: add network aware discord tapBot url and hide section if link does not exist

maparr committed:
fix: replace tap account link with Discord link for Feedback page

brusherru committed:
fix: rendering rewards on Smesher screen even if it is turned off

brusherru committed:
fix: do not wait for "wallet opened" event to send some data to renderer

brusherru committed:
fix: throttle saving account data to avoid collisions on fs

brusherru committed:
fix: treat PostSetupStates Prepared and Paused also as `isSmeshing`

brusherru committed:
tweak: improve Smapp logging

brusherru committed:
fix: add PAUSED state into `isSmeshing` selector

brusherru committed:
fix: make it possible to set up smeshing against standalone network

brusherru committed:
fix: remove excessive retries

brusherru committed:
fix: do not drop net services on wallet activation

maparr committed:
feature: add type for Node Error to define message type and reaction on it.

maparr committed:
feature: add statuses and messages for Network screen in case of not installed libs

brusherru committed:
refactor: run single retry loop for `isNodeAlive` at same time and pass Promise reference to others

maparr committed:
feature: add error message for specific node not specified error messages

brusherru committed:
fix: get rid of excessive restarts of NetService

brusherru committed:
fix: remove excessive update (it is a part of "StartupFlow")

brusherru committed:
fix: wait for Node ready before retrieving rewards

maparr committed:
feature: add checklist modal

brusherru committed:
fix: get rid of excessive handler on wallet change

brusherru committed:
fix: run GRPC Services for local node by default

brusherru committed:
Revert "fix: get rid of excessive handler on wallet change"

This reverts commit cabb672b5521706b7114e72621725e17628e133e.

brusherru committed:
fix: disable RESUME button on clicking it

brusherru committed:
fix: allow to set up smeshing before node syncs (for standalone)

brusherru committed:
fix: activate wallet even if it is the same everything (otherwise we may stuck on loading screen)

brusherru committed:
fix: Start NodeEvents and make condition more robust

brusherru committed:
fix: smesher screen

brusherru committed:
tweak: Remove excessive retries wrapper

brusherru committed:
fix: smesher screen (do not wait for `$isNodeReady`)

brusherru committed:
tweak: display actual error message on initFailed NodeEvent

brusherru committed:
Release v1.1.7

dependabot[bot] committed:
chore(deps-dev): bump electron from 20.3.11 to 26.2.0

Bumps [electron](https://github.com/electron/electron) from 20.3.11 to 26.2.0.
- [Release notes](https://github.com/electron/electron/releases)
- [Changelog](https://github.com/electron/electron/blob/main/docs/breaking-changes.md)
- [Commits](https://github.com/electron/electron/compare/v20.3.11...v26.2.0)

---
updated-dependencies:
- dependency-name: electron
  dependency-type: direct:development
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

dependabot[bot] committed:
chore(deps-dev): bump @babel/core from 7.21.8 to 7.22.17

Bumps [@babel/core](https://github.com/babel/babel/tree/HEAD/packages/babel-core) from 7.21.8 to 7.22.17.
- [Release notes](https://github.com/babel/babel/releases)
- [Changelog](https://github.com/babel/babel/blob/main/CHANGELOG.md)
- [Commits](https://github.com/babel/babel/commits/v7.22.17/packages/babel-core)

---
updated-dependencies:
- dependency-name: "@babel/core"
  dependency-type: direct:development
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

brusherru committed:
chore: turn off logging in console for dev

brusherru committed:
chore: remove excessive console.log

brusherru committed:
fix: do not overwrite stream subscriptions but accumulate them

maparr committed:
fixup! feature: add network aware discord tapBot url and hide section if link does not exist

brusherru committed:
fix: displaying rewards from stream api on smesher screen

brusherru committed:
tweak: add log messages & optimize sending node status

brusherru committed:
feat: add `debug:addViewAccount` script

brusherru committed:
chore: update yarn.lock

brusherru committed:
fix: resubscribe to rewards stream when wallet is unlocked

brusherru committed:
Release v1.1.8

maparr committed:
feature: handle user feedback form by sentry sdk and collect node and build version

brusherru committed:
Release v1.1.9

maparr committed:
Feature: push os system info to view and update error check list modal

maparr committed:
fix: remove left panel

maparr committed:
fix: light theme

brusherru committed:
chore: update packages to the latest dependabot proposals

maparr committed:
fix: update traces sample rate type by doc

brusherru committed:
fix: request for PoST providers only on a proper setup screen

brusherru committed:
Release v1.1.10

maparr committed:
Release v1.1.11

maparr committed:
chore: update deps for @sentry/types

brusherru committed:
feat: update node & npm engines

brusherru committed:
chore: update node version in CI

brusherru committed:
chore: specify engines strictly

maparr committed:
chore: update check disk space

brusherru committed:
fix: use virtual scroll for rendering rewards on smesher screen

maparr committed:
fix: resolve logical issue and show modal for lib issue or for unspecified issues from node

brusherru committed:
Release v1.1.12

brusherru committed:
tweak: add original config file hash as queryStr in requests for the config file

brusherru committed:
chore: update yarn.lock

brusherru committed:
Release v1.2.0

brusherru committed:
fix: calculate maxGas properly

brusherru committed:
feat: display node's startup status on network screen

brusherru committed:
test: update tests

brusherru committed:
chore: replace require with import

brusherru committed:
chore: update yarn.lock

brusherru committed:
Release v1.2.1

brusherru committed:
tweak: disable setup smeshing button while Node is offline

maparr committed:
fix: exception during switch theme

brusherru committed:
refactor: remove useless code

brusherru committed:
chore: downgrade electron to v25.8.4

maparr committed:
fix: remove store key and use auto-launch internal isEnabled fn

maparr committed:
fix: update CI/CD macos yarn test wrong arch run

brusherru committed:
refactor: unsubscribe from `PostSetupStatusStream` once it is Complete

brusherru committed:
fix: reduce logging for $rewards updates

brusherru committed:
fix: retrieve account balance until it will be finished successfully

monikasmolarek committed:
removed the throttle on pos generation if pc in use

monikasmolarek committed:
fix & remove leftovers

monikasmolarek committed:
visual adjustment

brusherru committed:
Release v1.2.2

brusherru committed:
Revert "fix: remove store key and use auto-launch internal isEnabled fn"

This reverts commit b9f77de50514d6fd905851c6def94e051bfff1ab.

brusherru committed:
fix: sync autostart value from config with OS on start and handle toggling properly

brusherru committed:
fix: log errors correctly

brusherru committed:
tweak: do not use storeService for autostart value

monikasmolarek commented:
pr was too far behind, rebasing it was complicated and I prefer restart it
